backend/inspirehep/disambiguation/tasks.py

```diff
@@ -64,3 +68,35 @@ def disambiguate_signatures(self, clusters):
             disambiguation_assigned_clusters.labels("2+").inc()
             LOGGER.debug("Received cluster with more than 1 author.")
     db.session.commit()
+
+
+@shared_task(ignore_result=False, bind=True)
+def disambiguate_authors(self, record_uuid):
+    record = InspireRecord.get_record(record_uuid)
+    if "Literature" not in record["_collections"]:
+        return
+    authors = list(record.get_modified_authors())
+    updated_authors = []
+    for author in authors:
+        if not author.get("curated_relation"):
```

Comment:
minor: it's almost always better to write this kind of stuff as
```suggestion
        if author.get("curated_relation"):
            continue
        # do the matching
```
to avoid having to indent so much, and make it less error-prone when extending the function.